pages/radio.js

```diff
@@ -1,15 +1,37 @@
+import { useState, useEffect } from 'react'
 import Layout from '../components/MainLayout';
 import MetaHead from '../components/MetaHead';
 import ChapterNavigation from '../components/ChapterNavigation';
 import ChapterTitle from '../components/ChapterTitle';
 import RadioData from '../data/radio';
+import { FaVolumeUp } from 'react-icons/fa'
 import { BASE_PATH } from '../constants';
 
 const title = 'Kode Radio Lengkap | Buku Saku Pramuka Digital';
 const desc = 'Daftar lengkap kode radio';
 const url = BASE_PATH + '/radio/';
 
 function LambangPramukaPage() {
+
+  const [canSpeak, setSpeak] = useState(false);
+
+  useEffect(() => {
+    // check if browser support Speech API
+    if('speechSynthesis' in window){
+      setSpeak(true) 
```

Comment:
can we use this as a reusable hooks @raisoturu ?
so we can use it on other page easier.

Reply:
I expect we just need to use like this:

```js
const { canSpeak } = useSpeachAPI();
```

Reply:
ah sure...
where I can put my hooks? 
Should I create a new directory "/utils/hooks.js"?

Reply:
just put on dir `/hooks`, no need to add `utils`

Reply:
okay

Reply:
Can you check my last commit? @mazipan